backend/src/main/resources/application-local.yml

```
@@ -4,22 +4,17 @@ spring:
     console:
       enabled: true
       path: /h2-console
       settings:
         web-allow-others: true
+
   data:
     redis:
       host: localhost
       port: 6379
+      username: default
       password:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)